runner/src/indexer-logger/indexer-logger.test.ts

```diff
@@ -127,5 +128,21 @@ describe('IndexerLogger', () => {
 
       expect(query).not.toHaveBeenCalled();
     });
+
+    it('log status for indexer', async () => {
```

Comment:
```suggestion
    it('writes status for indexer', async () => {
```
`log` confused me here since we also log 😅

Reply:
Oops sorry haha. Will re-title both tests.